asterius/rts/rts.gc.mjs

```diff
@@ -175,7 +175,7 @@ export class GC {
               break;
             }
             default:
-              throw new WebAssembly.RuntimeError();
+              throw new WebAssembly.RuntimeError("unhandled closure type: " + type);
```

Comment:
I recommend using JS template literals as demonstrated in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals. It looks nicer than string concats.